### ISSUE TITLE
Podivné zobrazení náhledů

### ISSUE THREAD

**Issue by nezbedova** [proarc/proarc#1524]
Malé náhledy se mi zobrazují podivně. Zkoušela jsem Firefox a Chrom.

![obrazek](https://user-images.githubusercontent.com/13831946/210760904-278b4892-5f0e-43e6-86c9-c4fdea3e2ffc.png)

Vypadá to, jako kdyby tam byl 4x a ořízlo se to se zarovnáním k levému hornímu rohu.

V jádru to vypadá takto:

![obrazek](https://user-images.githubusercontent.com/13831946/210761260-67673143-19f8-4df2-8c1f-2dc041d9dbaf.png)

Máte to takto ještě někdo?

**Comment by luckajirku** [comment]
Jo, to dělá od té doby, co tam máme ty extra složky. A neděje se to u všech dávek. Takže jsme se rozhodli to zatím ignorovat:-).

**Comment by ZdenkaSera** [comment]
Zkoušela jsem souběžně s Martinou a taky se mi to zobrazuje jen u stt dávky, jinak ok.

**Comment by nezbedova** [comment]
@luckajirku Děkuji.

**Comment by ZdenkaSera** [comment]
@ZdenkaSera vyzkoušet s příštím releasem
